app/src/main/java/woowacourse/omok/repository/StoneRepository.kt

```diff
@@ -0,0 +1,41 @@
+package woowacourse.omok.repository
+
+import android.content.ContentValues
+import android.content.Context
+import android.provider.BaseColumns
+import domain.stone.Stone
+
+class StoneRepository(context: Context) {
+    private val db = StoneDbHelper(context).writableDatabase
```

Comment:
```suggestion
class StoneRepository(private val db: SQLiteDatabase) {
```
context는 SQLiteDatabase를 만들기 위해서만 사용하고 있어요.
그렇다면, SQLiteDatabase를 캡슐화하도록 개선해보면 어떨까요?